chapters/inheritance.tex

```diff
@@ -1022,7 +1026,24 @@ \subsection{Annotations for Redeclaration and Modification}\label{annotation-cho
   annotation(choicesAllMatching = true);
 \end{lstlisting}
 
-It can also be applied to nonreplaceable declarations, e.g.\ to describe enumerations.
+The \lstinline!choicesAllMatching=true! annotation can also be applied to parameter records.
```

Comment:
Wouldn't it be better to split into smaller examples (together with rewording the beginning of the first one)?
(Github won't let me suggest to remove the empty line after the listing.)
```suggestion
\end{example}

\begin{example}
Applying the \lstinline!choicesAllMatching = true! annotation to parameter records.
```